packages/protocol/contracts-0.8/common/test/MockCeloToken.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity >=0.8.0 <0.9.0;
```

Comment:
Ditto with duplicated contracts, would try to extend `GoldTokenMock.sol` to use flexible pragma, if possible. I think it should be fine.

Reply:
`GoldTokenMock` imports the GoldToken, which is set to `0.5.13`. this causes compilation to fail.

modifying `GoldTokenmock` would require updating goldToken to 0.8 as well. seemed out of scope

Reply:
Fair enough. Would just leave a comment clarifying that when Celo contract gets moved to 0.8, then the mocks should be unified.